<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWinClient</class>
 <widget class="QWidget" name="CWinClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电机综合测试仪本地控制端V0.1</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="16,1,1,1,1,1,1,1" columnstretch="3,0,2,0">
   <item row="0" column="0" colspan="4">
    <widget class="QTreeWidget" name="TabDevices">
     <attribute name="headerDefaultSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>Port</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>上线时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出厂编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MAC地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>软件版本</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" rowspan="7">
    <widget class="QTreeWidget" name="TabFileList">
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>当前客户端口:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="EditCurrentPort">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="BtnGetClientFileList">
     <property name="text">
      <string>获取文件列表*</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="TextCurrentFile">
     <property name="text">
      <string>当前选定文件:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="EditCurrentFile">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>获取客户文件</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>发送本机文件</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>当前系统命令:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="EditCommand">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>ls</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="KeyCommand">
     <property name="text">
      <string>发送系统命令</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="TextUser">
     <property name="text">
      <string>当前登录帐号:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="EditUserName">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>sa</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" rowspan="3">
    <widget class="QPushButton" name="BtnLogin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="TextPassword">
     <property name="text">
      <string>当前登录密码:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="EditPassword">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>1234</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QTextBrowser" name="Text"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
